config/family_config.csv

```diff
@@ -2,7 +2,7 @@ Inheritor,Type,Asaba_Rank,descendant,blocked_by,Ignore,Asaba
 son,A,1,M,[],0,"['daughter','daughter_x2']"
 son_of_son,A,2,M,[],0,"['daughter_of_son', 'daughter_of_son_x2']"
 father,FA,3,,[],0,[]
-father_of_father,FA,4,,['father'],1,[]
+father_of_father,FA,0,,['father'],0,[]

```

Comment:
does changing this affect this anywhere else for his Asaba rank?

Reply:
Let me run some tests.

Reply:
changed and pushed